dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -271,6 +268,14 @@ KRB5_FILE_FOUND=CWIML4516E: The LdapRegistry component is configured to use a {0
 KRB5_FILE_FOUND.explanation=The configured file was successfully located.
 KRB5_FILE_FOUND.useraction=No action is required.
 
+# CWIML4517 already used
+
 IDMAP_INVALID_FORMAT=CWIML4518W: The {0} {1} value is malformed. The value must be a series of objectclass:attribute or *:attribute pairs, where each pair is separated by a semi-colon.
```

Comment:
The message says the pairs must be separated by a semicolon, but the examples "objectclass:attribute or *:attribute" are separated by colons.

Reply:
Updated